command/windows_service_install_test.go

```diff
@@ -0,0 +1,361 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package command
+
+import (
+	"bytes"
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"text/template"
+
+	"github.com/hashicorp/cli"
+	"github.com/hashicorp/nomad/helper/winsvc"
+	"github.com/shoenig/test/must"
+)
+
+func TestWindowsServiceInstallCommand_Run(t *testing.T) {
+	t.Parallel()
+
+	freshInstallFn := func(m *winsvc.MockWindowsServiceManager) {
+		srv := winsvc.NewMockWindowsService(t)
```

Comment:
are the `m.t.Helper()` calls down in the `MockWindowsService/Manager` methods sufficient to show helpful line numbers (i.e. relevant to a specific test case) if these assertions fail?  I've found nested helper-helper funcs to be tricky in this regard.